### ISSUE TITLE
Feature Request: Copy to clipboard

### ISSUE THREAD

**Issue by marcobiedermann** [joshbuchea/HEAD#130]
I suggest adding an option to copy the code snippets to your clipboard.
[Clipboard.js](https://github.com/zenorocha/clipboard.js) is a nice JavaScript library which makes it super easy.
If you agree, I will start working on this :)

**Comment by joshbuchea** [comment]
@marcobiedermann apologies for not getting back to you on this sooner man. I haven't had much free time lately and have been neglecting my open source projects a bit. This sounds like a good idea to me 👍  My attempt to make it convenient to select/copy snippets by auto-selecting all text in a code block was a poor choice.

**Comment by marcobiedermann** [comment]
@joshbuchea Yeah, no worries. I was also very busy the last few weeks. Maybe I can work on this issue this weekend :)

**Comment by dakshshah96** [comment]
I don't think the "Copy to Clipboard" functionality is possible with the current layout of the static site, which is made on Jekyll. All the content on http://gethead.info is parsed from the `README.md` file. It is simply not possible to add the copy functionality in a markdown file.

One possible solution would be to divide every code block into individual files and place them in the `_includes` folder. It would then be possible to add a "Copy to Clipboard" button at the end of each code block insertion in the `index.html` file. This too, sounds too complicated to be implemented.

Did you have any other solution in mind @marcobiedermann?

cc: @joshbuchea

**Comment by joshbuchea** [comment]
@marcobiedermann did you have any interest in doing this? No worries at all if the answer is no, this issue was opened a LONG time ago.

If you have any interest at all, I'm happy for you to implement it. If you don't for any reason, I may let Copilot give it a try.

**Comment by mattsafaii** [comment]
@joshbuchea I'll take this on.
